iroha_network/src/lib.rs

```diff
@@ -239,6 +413,9 @@ mod tests {
 
     #[async_std::test]
     async fn single_threaded_async() {
+        #[cfg(feature = "mock")]
```

Comment:
I don't think that we need to mock network in network tests - we don't test anything in this case?

Reply:
We need to test the mock implementation and especially if its compatible with the main one. So this test (when the mock feature is enabled) ensures the mock works and has the same interface.

Reply:
oh, I missed the real tests - ok